pkg/msr/msr_linux.go

```diff
@@ -75,6 +77,7 @@ func Read(m []string, addr uint32) ([]uint64, []error) {
 	return regs, nil
 }
 
+// Write writes to a slice of msrs
```

Comment:
What's a slice of msrs?

Reply:
there's probably better ways to word this, eh? I just couldn't get  to one. 
If we think of the glob for a set of mrs, it produces the []string, which I thought of as a slice of msr's. What is a better way to say this?

Reply:
Done, PTAL